Phindr3D-Python/src/Clustering/Clustering.py

```diff
@@ -148,7 +158,7 @@ def pie_slice(prev_ratio, cur_ratio):
                 parts=np.array(pt[parts_ind[0].astype(int)])
                 for x in range(len(parts)):
                     s1, mark=pie_slice(sum(parts[:x]), sum(parts[:x+1]))
-                    self.main_plot.axes.scatter(plot_data[0][cluster], plot_data[1][cluster], marker=mark, s=s1 ** 2 *4000*rsize[size_ind], facecolor=colors[parts_ind[0][x]])
+                    self.main_plot.axes.scatter(plot_data[0][cluster], plot_data[1][cluster], marker=mark, s=s1 ** 2 *3000*rsize[size_ind], facecolor=colors[parts_ind[0][x]])
```

Comment:
Where does the 4000/3000 factor come from? Can we define it in the constructor or somewhere easier to find?

Reply:
factor defined as variable with comment.
self.max_piesize=3000 # maximum matplotlib piechart size.